code/espurna/ir.ino

```diff
@@ -14,7 +14,7 @@ Copyright (C) 2017-2018 by François Déchery
 
 IRrecv * _ir_recv;
 decode_results _ir_results;
-
+unsigned long last = millis();
```

Comment:
`last` is global variable for the whole espurna codebase. notice how other ones have `_ir` prefix